packages/HorizonUI/Sources/HorizonUI/Sources/Components/ProgressBar/HorizonUI.ProgressBar.swift

```diff
@@ -88,7 +88,7 @@ extension HorizonUI.ProgressBar {
                             .frame(minWidth: width, alignment: .trailing)
                     }
                 }
-                .clipShape(Capsule())
+                .huiCornerRadius(level: .level28)
```

Comment:
Please remove this `level28` corner radius as it's not part of the design system. Instead use `level6`. [Link](https://instructure.slack.com/archives/C081ZRVMCSG/p1734108990159199?thread_ts=1734100129.630279&cid=C081ZRVMCSG) to discussion.